.github/workflows/CI.yml

```
@@ -53,6 +53,13 @@ jobs:
           version: ${{ matrix.julia-version }}
       - uses: julia-actions/cache@v1
       - uses: julia-actions/julia-runtest@v1
+      - uses: julia
+      - name: Configure new FFMPEG version environment
+        shell: julia --project="." --color=yes {0}
+        run: |
+          using Pkg
+          Pkg.add(PackageSpec(url="https://github.com/theogf/FFMPEG.jl/tree/tgf/update-FFMPEG"))
+          Pkg.instantiate()
```

I presume you want to install FFMPEG _before_ running the tests, no?
```suggestion
      - name: Configure new FFMPEG version environment
        shell: julia --project="." --color=yes {0}
        run: |
          using Pkg
          Pkg.add(PackageSpec(url="https://github.com/theogf/FFMPEG.jl/tree/tgf/update-FFMPEG"))
          Pkg.instantiate()
      - uses: julia-actions/julia-runtest@v1
```

---

Indeed! I thought I did sorry.